### DISCUSSION TITLE
Juegos de guerra

### DISCUSSION THREAD

**Discussion by azagaya** [gamerevar/proyecto-comunitario-1-gd#60] [Archivados]
Podrian ser dos juegos de guerra, uno de estrategia, y el otro un shooter topdown o side scroller. Ponele que en uno jugas lo "macro" de la guerra, con unidades, construcciones (lo mas acotado posible), y el otro es un heroe especie comando que hace misiones, en la misma guerra, pero en paralelo. Entonces, una misión en el juego de estrategia puede ser sobrevivir hasta que llegue el heroe, y en el otro juego hay un level en que por tiempo tenes que llegar a la base a ayudar.

O al pasar un nivel en el de estrategia se desbloquea un avance cietífico que te deja hacer nuevas unidades, o mejorar la resistencia de algo, y en el otro juego una de las misiones es robar el avance tecnológico de una base enemiga

Uno mostraría lo macro de la guerra, el otro lo mas personal

**Comment by LeandroCuello** [comment]
Uyyy me gusta mucho la idea, osea los juegos estaría muy conectados entre si, lo cual hace que nesecite más comunicación, pero es muy único y creo que puede valer la pena.
También se puede llegar a contar una historia interesante mostrando como vale más la vida de una sola persona que la de varias.

**Comment by jh62** [comment]
> Podrian ser dos juegos de guerra, uno de estrategia, y el otro un shooter topdown o side scroller. Ponele que en uno jugas lo "macro" de la guerra, con unidades, construcciones (lo mas acotado posible), y el otro es un heroe especie comando que hace misiones, en la misma guerra, pero en paralelo. Entonces, una misión en el juego de estrategia puede ser sobrevivir hasta que llegue el heroe, y en el otro juego hay un level en que por tiempo tenes que llegar a la base a ayudar.
> 
> O al pasar un nivel en el de estrategia se desbloquea un avance cietífico que te deja hacer nuevas unidades, o mejorar la resistencia de algo, y en el otro juego una de las misiones es robar el avance tecnológico de una base enemiga
> 
> Uno mostraría lo macro de la guerra, el otro lo mas personal

Esta buena la idea del juego, pero creo q la idea de los personajes es que sean antagonicos. Como sugeririas que fuera en ese caso?

(igual no es **obligatorio**, pero creo que era la idea original de los gd)

**Reply by azagaya** [reply]
Ah, simplemente en mi idea no eran antagónicos jaja. No se la verdad como adaptar para ese caso...

**Reply by roverem** [reply]
El juego macro puede ser narrado y ponerte en el rol de un general.
El comando podría ser un enemigo histórico del general (pelearon en la misma guerra, son hermanos, cualquier excusa es buena).

Sería piola hacer que el jugador termine (o gané) el juego pero los desenlaces de las historias sean de perdida para cada uno de los personajes en sus respectivos juegos. Es decir que ganar el juego sería perder (por eso de que no hay ganadores en la guerra).

No soy guionista asique no sabría darle forma a esto, pero quizás a alguno le interesa desarrollarlo.

**Reply by azagaya** [reply]
Si, puede ser. Igualmente ahora solo había que definir el juego en grandes rasgos. Por eso basicamente dije dos visiones de una guerra. Uno de estrategia, y otro un shooter. Sobre el resto creo que van a trabajar los game designers, y luego, si hay, los guionistas.

**Reply by roverem** [reply]
Bueno, con el comentario solo quería decir que apoyo la idea. Le veo potencial

**Reply by jh62** [reply]
> Si, puede ser. Igualmente ahora solo había que definir el juego en grandes rasgos. Por eso basicamente dije dos visiones de una guerra. Uno de estrategia, y otro un shooter. Sobre el resto creo que van a trabajar los game designers, y luego, si hay, los guionistas.

Está joya la idea. Lo de antagonista es un detalle nomás. No es inamovible.

**Reply by azagaya** [reply]
@roverem si, esta perfecto el comentario! perdón si di a entender algo distinto con mi comentario. Es más, esta bien que comentes esas ideas asi las agarran los designers y elijen las que quieran. Mi comentario era mas que nada justificando por que no me explaye mucho en esas cosas.
